.golangci.yml

```diff
@@ -8,21 +11,37 @@ linters:
     - govet
     - errcheck
     - staticcheck
-    - gofmt
-    - goimports
     - revive
     - ineffassign
-    - typecheck
     - unused
-    - gosimple
     - misspell
     - nakedret
     - bodyclose
     - gocritic
     - makezero
     - gosec
+  settings:
+    staticcheck:
+      checks:
+        - all
+        - '-QF1008' # Allow embedded structs to be referenced by field
+        - '-ST1000' # Do not require package comments
+    revive:
+      rules:
```

Comment:
Turned off a few code comment rules. Not that they are wrong, but because we should turn them on intentionally rather than just because we're doing an upgrade.